vpr/src/draw/draw_xtoy.cpp

```diff
@@ -0,0 +1,571 @@
+/*draw_xtoy.cpp contains all functions that draw lines from one point to another.*/
```

Comment:
I'd rename this file.  I initially read it as being a "toy".  It is really drawing the edges between routing resource (rr) nodes, so a better name would be 
draw_rr_edges.cpp